# Debris Position
float32 debris_position_x
float32 debris_position_y
float32 debris_position_z

# Debris Velocity
float32 debris_velocity_x
float32 debris_velocity_y
float32 debris_velocity_z

# Debris Information
float32 debris_diameter
int32 time_of_detection
int16 object_count
